vm_deploy.sh

```diff
@@ -76,7 +76,8 @@ echo -e "\n\033[1;32m==== Transferring files to VM ====\033[0m\n"
 rsync -av -e "ssh -o StrictHostKeyChecking=no -i ./id_ed25519" --delete --exclude={'.git','.gitignore','id_ed25519','id_ed25519.pub','cloud-init.yaml','README.md','vm_deploy.sh','vm_destroy.sh','commands.txt'} $(pwd) $USER@$(multipass info nameless | grep IPv4 | awk '{ print $2 }'):/home/$USER
 
 echo -e "\n\033[1;32m==== Execute install scripts on VM ====\033[0m\n"
-ssh -o StrictHostKeyChecking=no -i ./id_ed25519 $USER@$(multipass info nameless | grep IPv4 | awk '{ print $2 }') 'cd meshellica && bash go_install.sh'
+ssh -o StrictHostKeyChecking=no -i ./id_ed25519 $USER@$(multipass info nameless | grep IPv4 | awk '{ print $2 }') 'cd meshellica/go && bash go_install.sh'
+ssh -o StrictHostKeyChecking=no -i ./id_ed25519 $USER@$(multipass info nameless | grep IPv4 | awk '{ print $2 }') 'cd meshellica/python && bash python_install.sh'
```

Comment:
is there a way to pass something to this script to decide which line to run?

Reply:
Like what we did yesterday?

```
install.sh
if [ "$1" == "go" ]
then
    ssh -o StrictHostKeyChecking=no -i ./id_ed25519 $USER@$(multipass info nameless | grep IPv4 | awk '{ print $2 }') 'cd meshellica/go && bash go_install.sh'
elif ......

bash install.sh go
```

Reply:
What about just one line with variables passed?

`cd meshellica/$LANG && bash $LANG_install.sh`